### DISCUSSION TITLE
🚀  Our journey to MVP: New features available!

### DISCUSSION THREAD

**Discussion by SebConejo** [mnfst/llm-gateway#130] [Announcements]
Hi everyone 👋,

A month ago, we shared our Proof of Concept (PoC) with you. We’re excited to let you know that we’re making steady progress towards our MVP, targeted for the end of July. As part of this journey, we’ve just [released an update](https://github.com/mnfst/manifest/releases/tag/4.0.0-alpha.4) that includes:

- **Filter by relation**: You can now filter your data by relation. [See the SDK doc](https://manifest.build/docs/javascript-sdk#crud-operations) - [See the API doc](https://manifest.build/docs/rest-api#filter-by-relations)
- **Timestamp**: You can now add the Timestamp property to your entities. [See how to do it here](https://manifest.build/docs/properties#timestamp)
- **API Doc**: An API doc is now available.
  
Give it a try and let us know what you think on our [Discord feedback channel](https://discord.gg/FepAked3W7)!

Thanks for being a part of this journey with us!

Best, The [Manifest](https://manifest.build/) Team

**Comment by brunobuddy** [comment]
Closing this out. It's from Manifest's old backend-as-a-service days, which we don't work on anymore since the pivot. See #742 for where things are now.
